rubin_sim/scheduler/detailers/short_survey.py

```diff
@@ -18,10 +18,15 @@ class Short_expt_detailer(Base_detailer):
         The short exposure time to use.
     nobs : float (2)
         The number of observations to try and take per year
+    night_max : float (None)
+        Do not apply any changes to the observation list if the current night is greater than night_max.
+        XXX--note, in theory, detailers should probably be expanded to include their own basis functions.
```

Comment:
Can you move this comment out of the documentation that shows up in the compiled documentation and maybe into a # comment in the code (it seems more like a reminder for development than a comment to communicate to users)?
This might be worth bringing up in a more general idea -- how are detailers separate from surveys, what do they have in common, if they have basis functions should they also have other properties of surveys?